src/m365/aad/commands/o365group/o365group-recyclebinitem-clear.ts

```diff
@@ -0,0 +1,102 @@
+import { Cli,Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Group } from './Group';
+import request from '../../../../request';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  confirm?: boolean;
+}
+
+class AadO365GroupRecycleBinItemClearCommand extends GraphItemsListCommand<Group> {
+  public get name(): string {
+    return commands.O365GROUP_RECYCLEBINITEM_CLEAR;
+  }
+
+  public get description(): string {
+    return 'Clears all O365 Groups from recycle bin.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.confirm = typeof args.options.confirm !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+
+    if (args.options.confirm) {
+      this.clearO365GroupRecycleBinItems(logger,args,cb).then(()=>{
```

Comment:
We should wrap this call in a function that we can call both here and in line 52 to avoid repeating the same code. See https://github.com/pnp/cli-microsoft365/blob/df6225ca6dcf80aa7fd90c15e36c147c033ee427/src/m365/aad/commands/approleassignment/approleassignment-remove.ts#L46 for example

Reply:
Got it. Fixed. Although I have kept the function itself outside of the `commandAction`